lamarr_virtual/roster.json

```diff
@@ -9,8 +9,12 @@
       }
     ]
   },
+  
   "students": [
     {
+      "name": "Kyrylo Rybalko",
+      "github": "kay-dev-uk"
+    },
       "name": "Mohamed Abdalla",
       "github": "Mohakim231"
     },
```

Comment:
Be careful, I think you removed an opening curly brace by mistake, there should be one between line 17 and 18 :)